deployments/.env.template

```
@@ -8,8 +8,8 @@ DB_PASSWORD=''
 DB_USER=''
 DOMAIN='0.0.0.0'
 NODE_ENV='development'
-PUBLIC_API_URL='http://localhost:3000'
-PUBLIC_APP_BASE_URL='http://localhost:5173'
+PUBLIC_API_URL='https://api.hoalu.localhost'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)